station/models.py

```diff
@@ -23,151 +24,130 @@
 BASIN_IMAGE_PATH = "station/basin_image/"
 BASIN_FILE_PATH = "station/basin_file/"
 
+User = get_user_model()
 
-class Country(models.Model):
-    """
-    The country that a station or region is in.
-    """
+
+class StationBaseModel(models.Model):
+    owner = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
+
+    PERMISSIONS_LEVELS = [
+        ("public", "Public"),
+        ("private", "Private"),
+    ]
+
+    permissions_level = models.CharField(
+        max_length=8, choices=PERMISSIONS_LEVELS, default="private"
+    )
 
     id = models.AutoField("Id", primary_key=True)
     name = models.CharField(max_length=32)
 
     def __str__(self):
         return str(self.name)
 
+    class Meta:
+        abstract = True
+        ordering = ("id",)
+
+
+class Country(StationBaseModel):
+    """
+    The country that a station or region is in.
+    """
+
     def get_absolute_url(self):
         return reverse("station:country_detail", kwargs={"pk": self.pk})
 
-    class Meta:
-        ordering = ("id",)
+    class Meta(StationBaseModel.Meta):
```

Comment:
I've never seen this. Is it really necessary to inherit from the base class meta? For some reason, I thought this was picked automatically.